src/rt/lifetime.d

```diff
@@ -133,7 +133,7 @@ extern (C) void _d_delinterface(void** p)
 
 
 // used for deletion
-private extern (D) alias void function (Object) fp_t;
+private extern (D) alias void function (Object) @nogc fp_t;
```

Comment:
This is the type of the class's destructor.  `ClassInfo.destructor` is cast to this type in `rt_finalize2`.  But, what if the class's destructor is *not* attributed with `@nogc`?